fetch/handler.go

```diff
@@ -209,28 +212,45 @@ func (h *handler) handleMeshHashReq(ctx context.Context, reqData []byte) ([]byte
 		h.logger.WithContext(ctx).With().Warning("failed to parse mesh hash request", log.Err(err))
 		return nil, errBadRequest
 	}
-	lids, err := iterateLayers(&req)
-	if err != nil {
+	if err := h.validateReq(req); err != nil {
 		h.logger.WithContext(ctx).With().Debug("failed to iterate layers", log.Err(err))
 		return nil, err
 	}
-	if len(lids) > 0 {
-		hashes, err = layers.GetAggHashes(h.cdb, lids)
-		if err != nil {
-			h.logger.WithContext(ctx).With().Warning("failed to get mesh hashes", log.Err(err))
-			return nil, err
-		}
+	hashes, err = layers.GetAggHashes(h.cdb, req.From, req.To, req.By)
```

Comment:
What is the meaning of an aggregated hash?

Reply:
But why do we call them *aggregated*hash ?

Reply:
Don't know but changing it is not part of this PR 🙂